pages/report/index.tsx

```diff
@@ -25,11 +26,12 @@ const Report: NextPage = () => {
                     message={reportsResponse.errorMessage}
                 />
             )}
-            {reportsResponse.isSuccess && (
+            {reportsResponse.isSuccess && customersResponse.isSuccess && (
```

Comment:
You could add loading and error handling for `customersResponse`. It could simply be identical to [this](https://github.com/three-consulting/epoc-frontend/blob/main/pages/index.tsx#L39-L66).